resources/user-guides/attach-data.md

```diff
@@ -308,24 +313,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target parameters to Target location-displayed events
+### Attaching additional data to Target location-displayed events<a name="Target-Attach-Data-Location-Displayed"></a>
```

Comment:
Please update the heading as follows:
"Attach additional data to Target location-displayed events"